com.servoy.extensions/src/com/servoy/extensions/plugins/rawSQL/RawSQLProvider.java

```diff
@@ -431,7 +432,8 @@ public boolean js_notifyDataChange(String serverName, String tableName, IDataSet
 			plugin.getClientPluginAccess().getDatabaseManager().notifyDataChange(
 				DataSourceUtils.createDBTableDataSource(serverMapping.localServername, tableName), pksDataset, action);
 			// notifySelf = false so that the client id is sent as well (needed for tenant id)
-			return getSQLService().notifyDataChange(plugin.getClientPluginAccess().getClientID(), false, serverMapping.remoteServername, tableName, pksDataset,
+			return getSQLService().notifyDataChange(plugin.getClientPluginAccess().getClientID(), notifySelf ? notifySelf : false, serverMapping.remoteServername,
```

Comment:
why "notifySelf ? notifySelf : false," this is already a boolean so this can just be "notifySelf"